kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/utils/TestUtils.scala

```diff
@@ -848,10 +848,14 @@ trait AbstractTestUtils
   }
 
   def executeCrcSimple(result: TransactionCommitResult, engine: Engine): TransactionCommitResult = {
```

Comment:
The semantic this method wanted was: write the CRC file via simple mode. Instead, presently, the semantic it was implementing was: write the CRC file if the SimpleCrcHook was present. That was incorrect.